### ISSUE TITLE
some troubles while resume training

### ISSUE THREAD

**Issue by BingzheWu** [zhreshold/mxnet-ssd#80]
Hi, I have trained my own datasets using these ssd implementation . I have
some troubles with resuming training.  When I end training at 100 epoch, i got
the train object accuracy at 0.30, but when I resume the traing from 100 epoch, after 4 batches,
I suddenly got the accuracy at 0.99, even when I adjust the lr to 0.0. I infer that this is caused by
the optimizer init method. Have you every encountered this problem?

**Comment by zhreshold** [comment]
I think there's no accuracy measurement in the new release, what do you mean by acc at 0.3 or 0.99?

**Comment by BingzheWu** [comment]
> I think there's no accuracy measurement in the new release, what do you mean by acc at 0.3 or 0.99?

Thanks for your reply. I mean the Train-objectAcc, and when I use the resume option, I can not control the learning rate which the optimizer use.

**Comment by zhreshold** [comment]
In the new version, there's no more Train-objectACC, but you can always control the behavior of optimizer in train/train_net.py
